frontend/containers/DocumentEdit/DocumentEdit.jsx

```diff
@@ -218,7 +218,11 @@ class DocumentEdit extends Component {
       (documentId !== document.remote._id)
     const needsFetch = !document.remote || remoteDocumentHasChanged
 
-    if (isIdle && needsFetch && this.currentCollection && state.api.apis.length > 0) {
+    if 
+      (isIdle &&
+      needsFetch &&
+      this.currentCollection && state.api.apis.length > 0
+    ) {
```

Comment:
Could we format this as:

```
if (
  isIdle &&
  needsFetch &&
  this.currentCollection &&
  state.api.apis.length > 0
) {
```

Reply:
Sure. I'd prefer to indent once more - see next changes